tox.ini

```diff
@@ -6,7 +6,7 @@ deps =
     --requirement={toxinidir}/requirements/production.txt
 passenv = PIP_WHEEL_DIR PIP_FIND_LINKS PIP_NO_INDEX
 commands =
-    coverage run -p {envbindir}/trial --rterrors {posargs:mimic}
+    coverage run --source mimic --branch --parallel-mode {envbindir}/trial --rterrors {posargs:mimic}
```

Comment:
This limits the coverage report to only the mimic modules, included branch coverage and significantly speeds up the tests.